### PR TITLE
Fixes #33 - plug and play launch file for the ronex

### LINKED ISSUE #33: Plug and play
Shouldn't have to edit the launch file/urdf to start the ronex out of the box.

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

**Comment by ugocupcic** [comment]
do #61 first

File: sr_ronex_controllers/scripts/load_passthrough_controllers.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+        @param ronex_ids the ids of the ronexes
+        """
+        #building a list of controller names
+        controllers_list = []
+        for ronex_id in ronex_ids:
+            controllers_list.append("/ronex_"+ronex_id+"_passthrough")
+
+        #calling the services to load and switch the controllers on
+        rospy.wait_for_service('/pr2_controller_manager/list_controllers')
```
Comment (toliver) [inline sr_ronex_controllers/scripts/load_passthrough_controllers.py]: This shouldn't be an absolute name. We might have a case where we have 2 realtime loops and therefore 2 controller_managers.

File: sr_ronex_controllers/scripts/load_passthrough_controllers.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    def load_and_start_ctrl(self, ronex_ids):
+        """
+        Load and start the passthrough controllers for the list of given ronexes.
+
+        @param ronex_ids the ids of the ronexes
+        """
+        #building a list of controller names
+        controllers_list = []
+        for ronex_id in ronex_ids:
+            controllers_list.append("/ronex_"+ronex_id+"_passthrough")
```
Comment (toliver) [inline sr_ronex_controllers/scripts/load_passthrough_controllers.py]: Probably this doesn't need to be absolute either (although they are unique names).

File: sr_ronex_launch/launch/sr_ronex.launch [hunk 3]
```diff
@@ -11,8 +11,7 @@
   <rosparam command="load" file="$(find sr_ronex_controllers)/config/general_io_passthrough_controller.yaml" />
```
Comment (toliver) [inline sr_ronex_launch/launch/sr_ronex.launch]: This shouldn't be loaded anymore. We might keep it with some modifications as an example of how to define the params manually?

**Pull request merged**